### PR TITLE
Routing Shift

### PR DESCRIPTION
**Pull request by zjr**
Still to do (will make these into tasks or something):

- [ ] Routing directly to section is broken (i.e. can't link to `/work`).
- [ ] There's no navigation on subpages.
- [ ] Fix the inner scroll effects of sub pages.
- [ ] Fix Moony.
- [ ] Fix section header styles (line is broken).
- [ ] Fix vertical paging styles.
- [ ] Fix vertical paging buttons.
- [ ] Fix/enable "next" buttons.
- [ ] Enable homepage 'down' carat.
- [ ] Animate homepage 'down' carat?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,91 +1,43 @@
...
-                  {/*<Route path="/everdays" component={Project} />*/}
-                  {/*<Route path="/illustration" component={Project} />*/}
-                </Switch>
-              </CSSTransition>
-            </TransitionGroup>
-          )}
-        />
-      </Router>
+      <BrowserRouter className={appClasses}>
+        <>
```
Comment (spurkalurk) [inline src/App.js]: i didn't know you could do this

File: src/styles/index.scss [hunk 2]
```diff
@@ -16,22 +21,22 @@ body {
...
-    > div > div:last-child:not(.project-page) {
-      padding: 0 6rem;
-    }
-  }
+  //#root {
+  //  > div > div:last-child:not(.project-page) {
+  //    padding: 0 6rem;
+  //  }
+  //}
 }
```
Comment (spurkalurk) [inline src/styles/index.scss]: do you want to just delete the above commented out stuff?

File: src/App.js (hunk 1 above)
Comment (zjr) [reply src/App.js]: It's _newww_.

File: src/styles/index.scss (hunk 2 above)
Comment (zjr) [reply src/styles/index.scss]: Yeah, must have just missed.

**Pull request merged**